packages/core/src/components/header/test/default/header.e2e.ts

```diff
@@ -0,0 +1,35 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/header/test/default/index.html';
+
+test.describe('tds-header-default', () => {
+  test('renders default header correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const headerComponent = page.getByRole('navigation');
+    await expect(headerComponent).toHaveCount(1);
+
+    /** Check screenshot diff */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('Header title exists and is "Example: default"', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const headerComponentHeaderText = page.getByText('Example: default');
+    await expect(headerComponentHeaderText).toBeVisible();
+  });
+
+  test('Header luncher icon exists', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const headerComponentLuncherIcon = page.getByRole('img');
+    await expect(headerComponentLuncherIcon).toBeVisible();
+    await expect(headerComponentLuncherIcon).toHaveCount(1);
+  });
+
+  test('Header brand label with link exists', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const headerComponentBrandLink = page.getByRole('link');
```

Comment:
I guess this locator could unintentionally select links in the header-launcher as well if the implementation of that was be changed so that the launcher list is not removed from the DOM while it is hidden.

Reply:
Yes then this test changes and has to be updated by that changer of that logic?.

Reply:
Yes updated: [2d0bc34](https://github.com/scania-digital-design-system/tegel/pull/461/commits/2d0bc348527c6ace24db0439ac34cbdf3c0d206b)